fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1523,6 +1523,11 @@ private void updateCatalogAfterCommitted(TransactionState transactionState, Data
         for (TableCommitInfo tableCommitInfo : transactionState.getIdToTableCommitInfos().values()) {
             long tableId = tableCommitInfo.getTableId();
             Table table = db.getTable(tableId);
+            if (table == null) {
+                LOG.warn("updateCatalogAfterCommitted table is null, table: {}, db: {}-{}",
+                        tableId, db.getId(), db.getFullName());
+                return;
```

Comment:
If this is a multi-table transaction and other tables sill exists, is it ok to return directly here? How about replace the `return` with `continue`?